apps/api/src/app/change/usecases/get-changes/get-changes.usecase.ts

```diff
@@ -95,11 +95,19 @@ export class GetChanges {
     entityId: string,
     environmentId: string
   ): Promise<IViewEntity | Record<string, unknown>> {
-    const item = await this.notificationTemplateRepository.findOne({
+    let item = await this.notificationTemplateRepository.findOne({
       _environmentId: environmentId,
       _id: entityId,
     });
 
+    if (!item) {
+      const items = await this.notificationTemplateRepository.findDeleted({
+        _id: entityId,
+        _environmentId: environmentId,
+      });
+      item = items[0];
+    }
```

Comment:
Could we do here one query findWithDeleted?

Reply:
I am not able to find it used in the project...

Reply:
ah it is a part of the mongoose-delete package :)

Reply:
Yea sorry i needed to refer it is from it :)